conf/defaults.yaml

```diff
@@ -0,0 +1,32 @@
+# This is example of USM QE configuration file.
+
+# All options in `usmqepytest` section will be loaded during pytest start
+# so that it will be available as if it were specified directly in pytest.ini
+# (see usmqe/usmqeconfig.py for details).
```

Comment:
This comment should be dropped completely, replaced with short statement about this being a definition of default values of common options.